docs/security-privacy-questionnaire.md

```diff
@@ -0,0 +1,105 @@
+- Validation: The browser uses the information in the payment link URI to validate the payment method and ensure it's supported before prompting the user.
+- User Choice: By exposing the payment link information, the browser can present the user with a choice of available payment options, allowing them to select their preferred method.
+
+The privacy implications of this are discussed [in the explainer](./docs/explainer.md#privacy-considerations).
+
+> 02.  Do features in your specification expose the minimum amount of information
+>      necessary to implement the intended functionality?
+
+Yes it is. The only information exposed is the payment link URI itself.
+
+The URI should contain the essential details required for the browser and payment clients to:
+- Identify the payment method: The URI scheme indicates the specific payment system being used (e.g., UPI, Bitcoin).
+- Initiate the payment: The URI may include parameters like payee address, amount, and currency, which are necessary to start the transaction.   
+- Validate the payment method: The browser uses the URI to check if the payment method is supported before prompting the user.
+
+> 03.  Do the features in your specification expose personal information,
+>      personally-identifiable information (PII), or information derived from
+>      either?
+
+No. Although the payment link URI may include parameters like payee address, amount, and currency, none of them belongs to above categories.
+
+> 04.  How do the features in your specification deal with sensitive information?
+
+The feature does not deal with sensitive information.
+
+> 05.  Do the features in your specification introduce state
+>      that persists across browsing sessions?
+
+No. The feature operates in a stateless manner.
+
+> 06.  Do the features in your specification expose information about the
+>      underlying platform to origins?
+
+No.
+
+> 07.  Does this specification allow an origin to send data to the underlying
+>      platform?
+
+Yes. The specification allows origins to send data to the platform only within the well-defined context of payment initiation, with user consent and security measures in place.
+
+> 08.  Do features in this specification enable access to device sensors?
+
+No.
+
+> 09.  Do features in this specification enable new script execution/loading
+>      mechanisms?
+
+No.
+
+> 10.  Do features in this specification allow an origin to access other devices?
+
+No.
+
+> 11.  Do features in this specification allow an origin some measure of control over
+>      a user agent's native UI?
+
+Possibly. When a valid payment link is detected, the browser may present a native UI to the user, allowing them to choose their preferred payment method or confirm the transaction. In this sense, the origin indirectly triggers the display of this UI by providing the payment link.
```

Comment:
Please remove "native". You can use "user agent UI" instead.

Please use the word "user agent" instead of "browser".

Can we be more specific about the conditions required to show the user agent UI, e.g., the user has enabling this type of payment link by enrolling with their wallet?

Reply:
> Please remove "native". You can use "user agent UI" instead.

Done.

> Please use the word "user agent" instead of "browser".

Also modified other uses in this questionnaire. 

> Can we be more specific about the conditions required to show the user agent UI, e.g., the user has enabling this type of payment link by enrolling with their wallet?

Done.